lib/yargs-factory.ts

```diff
@@ -1704,7 +1704,7 @@ export class YargsInstance {
   >(
     builder: (key: K, value: V, ...otherArgs: any[]) => YargsInstance,
     type: T,
-    key: K | K[] | {[key in K]: V},
+    key: K | K[] | {[key in K]: V | undefined},
```

Comment:
These type changes are from https://github.com/yargs/yargs/pull/2393 and were necessary to get it to compile.